generators/java/spring-web/generator/src/main/kotlin/io/github/fomin/oasgen/java/spring/web/JavaSpringMvcServerWriter.kt

```diff
@@ -112,26 +113,45 @@ class JavaSpringMvcServerWriter(
                 } else {
                     ""
                 }
+                val headerDefinitions = operation.parameters()
```

Comment:
I don't see the reason to define header-parameters separately. They can be defined with the other parameters

Reply:
> I don't see the reason to define header-parameters separately. They can be defined with the other parameters

changed it